CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ version directory, and  then changes are introduced.
 
 ## [Unreleased]
 
```

Comment:
You need to revert that. There is some sort of bug that doesn't apply filters when actions are added as a PR. I'll open a ticket to GH.

Reply:
I opened a support request. Hopefully they'll get back to me.